routes/emails.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const Router = require('express').Router;
+const Message = require('../models/Message');
+
+const router = Router();
+
+router.get('/', (req, res) => {
+  res.status(200).end();
+})
+
+router.post('/', (req, res) => {
+  let message = new Message(req.body);
+  message.save(req.body, (err, message) => {
+    if (err) return err;
+    res.status(200).end();
```

Comment:
It's idiomatic REST to respond with the created resource.
`res.status(201).json(message);`